# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libmpeg3/largefile.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Needed for compilation on at least ppc64, maybe all architectures.

  - Rene Rebe <rene@exactcode.de>

--- libmpeg3-1.6/mpeg3io.c.vanilla	2006-06-04 13:23:42.000000000 +0200
+++ libmpeg3-1.6/mpeg3io.c	2006-06-04 13:26:14.000000000 +0200
@@ -1,6 +1,7 @@
 #include "mpeg3private.h"
 #include "mpeg3protos.h"
 
+#define __USE_LARGEFILE64
 #include <mntent.h>
 #include <stdint.h>
 #include <stdlib.h>
